### DISCUSSION TITLE
v1.1.7

### DISCUSSION THREAD

**Discussion by jp7677** [avisi-cloud/structurizr-site-generatr#350] [Announcements]
### Bugfixes

* Ignore case in branch name comparator by @qtzar in https://github.com/avisi-cloud/structurizr-site-generatr/pull/346
* Create a new diagram cache for each site generation by @jp7677 in https://github.com/avisi-cloud/structurizr-site-generatr/pull/349

**Full Changelog**: https://github.com/avisi-cloud/structurizr-site-generatr/compare/1.1.6...1.1.7

<hr /><em>This discussion was created from the release <a href='https://github.com/avisi-cloud/structurizr-site-generatr/releases/tag/1.1.7'>v1.1.7</a>.</em>

**Comment by jp7677** [comment]
@dirkgroot Could you please start the brew magic again? Thanks!

**Reply by dirkgroot** [reply]
Done 😄.
